middlewares/isValidParameterId.middleware.js

```diff
@@ -0,0 +1,14 @@
+import { isValidObjectId } from 'mongoose';
+import { ValidationError } from '../helpers/errors.js';
+
+export const isValidParameterId = (req, _res, next) => {
```

Comment:
we have it in another PR, I'd stick with that version after it will be merged

Reply:
Ok.